functions/placeShipsRandom.ts

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Ship } from "../interfaces/types";
+
+// x=col y=row
+// Add a thing to check for coordinate outside the grid
```

Comment:
comments for how a function works should be in JSDoc format like below:
/**

*/

Reply:
Why is this function outside of `Game.tsx`?

Reply:
I figured that I didn't want to populate the `Game.tsx` file, and the random generation can be handled outside - the generation should be at the initial stage of the game and won't be recalled later I suppose. Idk if that would cause any inconvenience but, I think `Game.tsx` will be populated soon with game logics.